tests/test_octodns_manager.py

```diff
@@ -1294,6 +1294,23 @@ def test_config_secret_handlers(self):
             requires_dummy.fetch(':hello', None),
         )
 
+    def test_zone_threshold(self):
+        with TemporaryDirectory() as tmpdir:
+            environ['YAML_TMP_DIR'] = tmpdir.dirname
+
+            manager = Manager(get_config_filename('zone-threshold.yaml'))
+
+            zone = manager.get_zone('unit.tests.')
+
+            self.assertEqual(0.2, zone.update_pcent_threshold)
+            self.assertEqual(0.1, zone.delete_pcent_threshold)
+
+            # subzone has different threshold
+            subzone = manager.get_zone('subzone.unit.tests.')
+
+            self.assertEqual(0.02, subzone.update_pcent_threshold)
+            self.assertEqual(0.01, subzone.delete_pcent_threshold)
+
```

Comment:
Probably good to get a test in here to make sure `zone.*pcent*` are `None` for things where it's not specified. Looks like that should be as simple as adding a 3rd zone to `zone-threshold.yaml` that doesn't specify a value.

Reply:
Makes sense; pushed that additional test.